example/app/api/serializers.py

```diff
@@ -30,17 +32,36 @@ class Meta:
 class TestModelDetailSerializer(
     saritasa_drf_tools.serializers.ModelBaseSerializer,
 ):
-    """List Serializer."""
-
-    related_model = RelatedTestModelSerializer(
-        read_only=True,
-    )
+    """Detail Serializer."""
 
     class Meta:
         model = models.TestModel
         fields = (
             "id",
             "text_field",
-            "related_model_id",
             "related_model",
         )
+        data_fields: typing.ClassVar = {
```

Comment:
"data fields" sounds too generic. What do you think about adding more DRF-related terms like
* nested
* depth

https://www.django-rest-framework.org/api-guide/serializers/#specifying-nested-serialization

?

Also, can we avoid `typing.ClassVar` usage for simplicity?